### ISSUE TITLE
Feature Request - Disable Summon Like Commands after a Game is Over

### ISSUE THREAD

**Issue by tvaduva** [mtgred/netrunner#7248]
Issue came up in #jinteki_net channel in mid-November that some players are seeing unknown cards by summoning cards to trash, reveal, or expose cards after a game is finished and opponents leave the game.

Of course, playing in a public game, you should expect that a lot of your decklist is learned through the regular game play. But, this seems to go beyond the expected if the opponent is no longer in the game. It seems like it could easily be limited if the opponent leaves the game or if game has already been decided.

**Comment by butzopower** [comment]
Not sure how much more challenging it would be, but maybe an alternative would be to limit `/summon` to only what's available in the lobby's card pool? This would at least limit some of the more egregious abuses which tend to require rotated cards.

Or maybe `/summon` should require a confirmation from the other player similar to `/undo-turn`.

Otherwise I'm not sure this really limits things all that much considering someone could just very quickly fire off all these commands during a mulligan if they were really determined.

**Comment by tvaduva** [comment]
Yeah, it's worth exploring. The thinking is that as long as the other player is aware, then if an opponent does shady things, they could be blocked or reported.

Limiting the command to cards in the cardpool would fluctuate it's usefulness depending on which cards are released and there are formats that use it (e.g. using a resource ability as a identity ability, etc.).

I do like the require confirmation of the other player as a way to limit the command. So that seems fine if it's easier/better to implement.

**Comment by tvaduva** [comment]
I'm not quite sure I understand why limiting to the game's legal cardpool would be a challenge to fix game states, but I will give you the benefit of the doubt there. That may not be a good solution, but the others suggested still seem like good options.

Please see this request in the spirit it was written. Yes, players should expect cards played to be public in a public game. But, this goes beyond that, it's allowing the revealing of information without the deck owner's knowledge. Whatever anyone may feel about gating information, I think they can see that taking that information should not be taken without the person's knowledge.

If the player wanted to know the rest of the decklist, they should ask for it. Or, by having matching commands like was suggested by [butzopower](https://github.com/butzopower) earlier.
